src/promptflow/promptflow/contracts/run_mode.py

```diff
@@ -6,13 +6,27 @@
 
 
 class RunMode(str, Enum):
+    """An enumeration of possible run modes.
+
+    :cvar Test: The test mode.
+    :cvar SingleNode: The single node mode.
+    :cvar Batch: The batch mode.
+    """
+
     Test = "Test"
     SingleNode = "SingleNode"
     Batch = "Batch"
 
     @classmethod
     def parse(cls, value: str):
-        """Parse string to RunMode."""
+        """Parse a string to a RunMode enum value.
+
+        :param value: The string to parse.
+        :type value: str
+        :return: The corresponding RunMode enum value.
+        :rtype: RunMode
```

Comment:
use `~promptflow.xxx.RunMode` to point to the class if this is our own class entity.

Reply:
Are you suggesting below modification?

```suggestion
        :rtype: promptflow.contracts.run_mode.RunMode
```

Reply:
Yes, but with `~` at the front. See 
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ml/azure-ai-ml/documentation_guidelines.md

![image](https://github.com/microsoft/promptflow/assets/2572521/cf1e9547-63ed-43ee-806e-55d5c8bb70d7)